azure-pipelines.yml

```diff
@@ -71,6 +71,10 @@ jobs:
         displayName: Set NuGet Version to PR Version
         condition: and(succeeded(), eq(variables['build.reason'], 'PullRequest'))
       # build, test and pack the packages
+      - task: CmdLine@2
```

Comment:
change the order to follow macOS pipeline.
Test Task restores all packages, builds dependent projects and then runs tests